### PR TITLE
Global configuration variables

### PR DESCRIPTION
**Pull request by baturozgur**
Added global configuration variables.
Added Kubernetes secret example.
Added Windows environment variable powershell command.
ED_ENV_VARS described for linux/mac service  installation.

Markdowns:
[Cofiguration Variables](https://github.com/eddocs/doc/blob/7819a568113c7e548de6f49f1c1f3d309714673e/configuration/variables.md)
[Environment  Variables](https://github.com/eddocs/doc/blob/7819a568113c7e548de6f49f1c1f3d309714673e/installation/environment-variables.md)

### CODE REVIEW

File: configuration/variables.md [hunk 1]
```diff
@@ -34,3 +40,16 @@ Instead of explicitly putting it into configuration it can be referred from agen
         endpoint: {{ Env "MY_SLACK_ENDPOINT" }}
 ```
 
+## Global Configuration Variables
+
+In certain scenarios it is useful to set a variable in a simple way. On production environment with large number of agents it is impractical to toggle certain features, change destinations or update parts of the processing rules by updating operating system variables.
+
+Configuration Variables table under [Global Settings](https://admin.edgedelta.com/global-settings) provides an easy way to set or override such variables. Variables will be replaced by their values in the table at the next agent heartbeat and agent will not try to acquire values from local agent environment.
```
Comment (taylanisikdemir) [inline configuration/variables.md]: Replace this sentence "Variables will be replaced by their values in the table at the next agent heartbeat and agent will not try to acquire values from local agent environment." with this:

"Variable updates will be detected automatically by the agents and will be in use within a minute"

File: installation/environment-variables.md [hunk 2]
```diff
@@ -57,18 +57,65 @@ description: >-
   </tbody>
 </table>
 
-## Examples - Linux
+## Examples - Kubernetes \(yml configuration\)
```
Comment (taylanisikdemir) [inline installation/environment-variables.md]: do we need to add "\" prefix for parantheses? I also saw bunch of them on processors page and removed but not sure.

File: installation/environment-variables.md [hunk 3]
```diff
@@ -57,18 +57,65 @@ description: >-
...
+      - name: ED_API_KEY
+        valueFrom:
+          secretKeyRef:
+            key: ed-api-key
+            name: ed-api-key
+      - name: HTTPS_PROXY
+        value: <your proxy details>
 
 ```
+### Examples - Kubernetes Secret \(yml configuration\)
```
Comment (taylanisikdemir) [inline installation/environment-variables.md]: the above kubernetes example also has secreto from environment. didn't understand the difference between these two versions of config. can we merge them and have just k8s example?

**Review by taylanisikdemir — APPROVED** [review]

File: installation/environment-variables.md (hunk 2 above)
Comment (baturozgur) [reply installation/environment-variables.md]: It was already there, removed it.

File: installation/environment-variables.md (hunk 3 above)
Comment (baturozgur) [reply installation/environment-variables.md]: Merged them.

File: configuration/variables.md (hunk 1 above)
Comment (baturozgur) [reply configuration/variables.md]: Done

**Pull request merged**